code/modules/mob/living/brain/MMI.dm

```diff
@@ -61,6 +61,9 @@
 			brainmob.stat = CONSCIOUS //we manually revive the brain mob
 			GLOB.dead_mob_list -= brainmob
 			GLOB.alive_mob_list += brainmob
+		if(!newbrain.brain_death && newbrain.damaged_brain && !newbrain.suicided && !brainmob.suiciding) // the brain is damaged, but not from a suicider
```

Comment:
since these two checks (l60 and l64) share all but one arg, condense the common ones into a var and check against that + the difference instead.

also else if

Reply:
What would be the best way to condense these vars; a proc that checks for all three and returns TRUE if they meet the condition?

Reply:
I think shiz wants something like var/isGoodBrain = newbrain.brain_death && newbrain.damaged_brain 

or something like that